Lib/_pyrepl/keymap.py

```diff
@@ -146,11 +150,11 @@ def _parse_key1(key, s):
                 meta = 1
                 s += 3
             elif c.isdigit():
-                n = key[s + 1 : s + 4]
+                n = key[s + 1: s + 4]
```

Comment:
that's misformatted. Those are not annotations. It was good before. Ditto all other slices you changed.

Reply:
Will fix, thanks 🙂